cpp/include/Ice/InputStream.h

```diff
@@ -859,13 +859,13 @@ class ICE_API InputStream : public IceInternal::Buffer
      * Reads a float from the stream.
      * @param v The extracted float.
      */
-    void read(Float& v);
+    void read(float& v);
 
     /**
      * Reads a sequence of floats from the stream.
      * @param v A vector to hold a copy of the float values.
      */
-    void read(std::vector<Float>& v);
+    void read(std::vector<float>& v);
```

Comment:
I find the param description odd. What about: An output vector filled by this function.